unit_test/batched/Test_Batched_TeamGemm.hpp

```diff
@@ -130,67 +130,220 @@ namespace Test {
         }
     EXPECT_NEAR_KK( diff/sum, 0, eps);
   }
+
+  template<typename DeviceType,
+           typename ViewType,
+           typename ScalarType,
+           typename ParamTagType, 
+           typename AlgoTagType>
+  void impl_test_batched_teamgemm_half(const int N, const int matAdim1, const int matAdim2, const int matBdim1, const int matBdim2,
```

Comment:
same thing again why the name if this is a generic function?

Reply:
same as above :)